meson.build

```diff
@@ -149,3 +149,14 @@ if cmocka_dep.found()
     test(name, executable(name, kwargs: t), suite: 'unit')
   endforeach
 endif
+
+# generate grout.pc
+grout_pkg = import('pkgconfig')
+grout_pkg.generate(
+  name        : 'grout',
+  description : 'Helpers for grout integration and management',
```

Comment:
I'm not sure about this description. Do we need it?

Reply:
pkgconfig.generate: if a library is not passed as a positional argument, the 'description' keyword argument is required.

Reply:
ack